src/models/streaming/SessionConfig.ts

```diff
@@ -18,18 +19,22 @@ export class SessionConfig {
      *      (ums, ahs) from transcript
      * @param deleteAfterSeconds (Optional) number of seconds after job completion
      *      when job is auto-deleted
+     * @param language (Optional) language identified using ISO 639-1 2-letter
+     *      language code
      */
     constructor(
         metadata?: string,
         customVocabularyID?: string,
         filterProfanity?: boolean,
         removeDisfluencies?: boolean,
-        deleteAfterSeconds?: number
+        deleteAfterSeconds?: number,
+        language?: string
```

Comment:
We don't want to expose the language parameter for streaming. This functionality doesn't exist yet.